app/routes/ready-reckoner/select-standard.js

```diff
@@ -0,0 +1,92 @@
+const standards = require('../../services/standards')
+const htmlContent = require('./select-standard-content')
+
+const Wreck = require('@hapi/wreck')
+const { agreementServiceBaseUrl } = require('../../config/general')
+
+const pageDetails = {
+  path: '/select-standard',
+  nextPath: '/selected-summary',
+  template: 'select-standard'
+}
+
+function getContentDetails (payload, selected, errorText = null) {
+  return {
+    title: 'Funding options you qualify for',
+    components: {
+      standards: {
+        idPrefix: 'standards',
+        name: 'standards',
+        hint: {
+          text: "Choose the options you want funding for. We'll pay you in monthly instalments so that work can begin without delay."
+        },
+        items: [
+          {
+            value: 'arable',
+            text: `Arable land, £${standards.arable.paymentRate} a hectare, plus £13 per tree`,
```

Comment:
I think `payload` should be used for acquiring the `paymentRate`, in the same way `userInput` and `payment`.